### ISSUE TITLE
关于train.py和predict.py

### ISSUE THREAD

**Issue by yyqAlisa** [bubbliiiing/yolov4-pytorch#170]
用trian.py文件跑
Total Loss: nan || Val Loss： nan
一直都是这样，请问是什么原因呢？

训练predict.py文件的时候，能输入图片img/street.jpg，但是没有出来预测结果，这又是为什么？

**Comment by bubbliiiing** [comment]
前一个我建议默认
后一个的话，好像没啥可以调的啊
